app/routes/account_.$accountId.create/route.tsx

```diff
@@ -46,24 +43,18 @@ export const loader: LoaderFunction = async ({ request, params }) => {
     return redirect(`/account/${params.accountId}`)
   }
 
-  const userRole = getUserAccountRole(
-    getUserAccountResponse.getUserAccount.users,
-    user.user.portalUserID,
-  )
+  const userAccount = getUserAccountResponse.getUserAccount
+  const userRole = getUserAccountRole(userAccount.users, user.user.portalUserID)
 
   if (!userRole || userRole === RoleName.Member) {
     return redirect(`/account/${params.accountId}`)
   }
-  const portalApps = getUserAccountResponse.getUserAccount.portalApps
+  const portalApps = userAccount.portalApps
   const underMaxApps = () => {
-    return !portalApps || portalApps.length < MAX_USER_APPS
+    return !portalApps || portalApps.length < userAccount.plan.appLimit
```

Comment:
is it worth extracting this out into a common function?

Reply:
Good call 🫡